openmetadata-docs/content/v1.5.x-SNAPSHOT/connectors/database/doris/index.md

```diff
@@ -65,6 +66,16 @@ serviceConnectionPath: "/images/v1.5/connectors/doris/service-connection.png",
 
 {% /stepsContainer %}
 
+## Securing Doris Connection with SSL in OpenMetadata
+
+To establish secure connections between OpenMetadata and Doris, navigate to the `Advanced Config` section. Here, you can provide the CA certificate used for SSL validation by specifying the `caCertificate`. Alternatively, if both client and server require mutual authentication, you'll need to use all three parameters: `ssl_key`, `ssl_cert`, and `ssl_ca`. In this case, `ssl_cert` is used for the client’s SSL certificate, `ssl_key` for the private key associated with the SSL certificate, and `ssl_ca` for the CA certificate to validate the server’s certificate.
+
+{% image
+  src="/images/v1.4/connectors/ssl_connection.png"
```

Comment:
should we use the images under `v1.5/connectors` here as we have the same images under `v1.5` as well ?

Reply:
Updated the path to v1.5